Multisig/Data/Services/Utils/JSONHTTPClient.swift

```diff
@@ -13,6 +13,8 @@ public protocol JSONRequest: Encodable {
     var urlPath: String { get }
     /// Query parameters
     var query: String? { get }
+    /// URL that overrides the urlPath and query
+    var url: URL? { get }
```

Comment:
I also added the url attribute in a past sprint before removing it again, I think in this case we can use this approch instead of PagedRequest.

Reply:
yes, that would work